nmdc_api_utilities/linked_instances.py

```diff
@@ -0,0 +1,96 @@
+# -*- coding: utf-8 -*-
+import requests
+from nmdc_api_utilities.nmdc_search import NMDCSearch
+
+
+class LinkedInstances(NMDCSearch):
+    """
+    Class for interacting with the NMDC linked instances API.
+
+    Inherits from NMDCSearch to utilize the base URL setup.
+    """
+
+    def __init__(self, env="prod"):
+        super().__init__(env=env)
+
+    def linked_instances(
+        self, types: list[str] | str, ids: list[str] | str
+    ) -> list[dict]:
+        """
+        Given a list of sample ids, find the associated study ids.
+
+        Parameters
+        ----------
+        types : list[str] | str
```

Comment:
Consider switching the order of these parameters and make type not required (so it would return all types with nmdc:NamedThing as the default).